databases/week2/sql_Week2Part1.sql

```diff
@@ -3,7 +3,7 @@
 USE hyf_lesson2;
 -- Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
 INSERT INTO task
-VALUES (36, 'Pick-up kids', 'Get the kids from schools', '2020-09-06 03:16:47', '2020-10-10 16:56:58', null, 2);
+VALUES ('Pick-up kids', 'Get the kids from schools', '2020-09-06 03:16:47', '2020-10-10 16:56:58', null, 2);
```

Comment:
Did you test this? I'm not sure it will work properly. Since you are not inserting all fields, you probably have to specify the ones that you are inserting, like this:

```
insert into task 
(title, description, created, updated, due_date, status_id) 
 values ('Pick-up kids', 'Get the kids from schools', '2020-09-06 03:16:47', '2020-10-10 16:56:58', null, 2);
```

Reply:
I must have missed on the testing that one out and I have no idea on how I missed it. Have made the corrections. Thanks a lot